.github/workflows/kubesec.yml

```diff
@@ -0,0 +1,58 @@
+name: Kubesec
+on:
+  push: 
+    branches: [ Add-kubesec-nam ]
+  schedule:
+    - cron: '22 8 * * 4'
+
+jobs:
+  setup:
+    runs-on: ubuntu-20.04
+    permissions:
+      actions: read
+      contents: read
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Collect all yaml
+        id: list_yaml
+        run: |
+          LIST_YAML="$(find * -type f -name '*.yaml')"
+          echo "::set-output name=value::$(IFS=$','; echo $LIST_YAML | jq -cnR '[inputs | select(length>0)]'; IFS=$'\n')"
+    outputs:
+      matrix: ${{ steps.list_yaml.outputs.value }}
+
+  lint:
+    needs: [ setup ]
+    name: Kubesec
+    runs-on: ubuntu-20.04
+    permissions:
+      actions: read
+      contents: read
+      security-events: write
+    strategy: 
+      matrix: 
+        value: ${{ fromJson(needs.setup.outputs.matrix) }}
```

Comment:
Have you considered putting all the files together in one big file instead of running separate jobs for them? I'm thinking it would take less time and resources probably, but maybe it would also be harder to tell where exactly the issue is when the scanner detects something.
No strong opinion here either, I just thought that 91 jobs is quite a lot :smile:

Reply:
I tried to make it in one single file. You can see it in the first commit. However, one technical issue is that one single file is generated while github action is running, not a part of the repo, so we cannot see file preview and determine where is the issue. 
In fact, we don't need to worry about 91 jobs can consume more time because they runs parallelly. Also, github allocates unlimited resource for open source repo to use github action, so we don't need to worry about result consumption.

Reply:
Nice! Thanks for the explanation!